tests/Checks.hs

```diff
@@ -2,6 +2,7 @@
 module Main where
```

Comment:
Yep, it never makes any practical difference... Just something I was thinking about given I just added the check to HLint.